fireworks_poe_bot/fw_poe_server_bot.py

```diff
@@ -193,7 +249,8 @@ async def get_response(
             return
 
     async def get_settings(self, setting: SettingsRequest) -> SettingsResponse:
-        """Override this to return non-standard settings."""
+        if "llava" in self.model:
```

Comment:
Maybe just make this a configuration flag in `args` and set it when we deploy to k8s?